build_image.sh

```
@@ -0,0 +1,9 @@
+#!/bin/bash
+oc create -f deploy/crds/neutron_v1_neutronovsagent_crd.yaml
+oc create -f deploy/crds/neutron_v1_neutronsriovagent_crd.yaml
+oc create -f deploy/crds/neutron.openstack.org_ovsnodeosps_crd.yaml
+oc create -f deploy/crds/neutron.openstack.org_ovncontrollers_crd.yaml
+make
+operator-sdk build --image-builder buildah "${BUILD_IMAGE}:${TAG}"
+sed -i 's|REPLACE_IMAGE|"${BUILD_IMAGE}:${TAG}' deploy/operator.yaml
+podman push --authfile "${REGISTRY_AUTH}" "${BUILD_IMAGE}:${TAG}"
```

should this script fail early if all those variables are not set?